lib/active_fulfillment/services/shopify_api.rb

```diff
@@ -86,11 +88,22 @@ def send_app_request(action, headers, data)
         end
       end
 
-      log :info, "GET response action=#{action} in #{"%.4fs" % realtime} #{response}"
-
+      log_response(action, realtime, response)
       response
     end
 
+    def log_response(action, realtime, response)
+      if response.try(:length).to_i > MAX_LOGGED_RESPONSE_LENGTH
```

Comment:
Didn't use `truncate = if` because policial wanted the `else` and `if` to be aligned which then makes it really hard to respect the 80 char line limit